### PR TITLE
Adding new keyboard - Split 89

### PR DESCRIPTION
**Pull request by jurassic73**
## Description

Adding new keyboard, the split89.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I ca

### CODE REVIEW

**Comment by jurassic73** [comment]
Missed the following:

readme.md - 
split89
should be
split89

**Review by jurassic73 — COMMENTED** [review]
Fixed issues in pull request.  Ready for review.

**Review by zvecr — CHANGES_REQUESTED** [review]
per <https://github.com/jurassic73/split89#bill-of-materials>, this needs to be relocated to the `keyboards/handwired/` folder

**Comment by jurassic73** [comment]
> per the images shown here, https://github.com/jurassic73/split89#bill-of-materials, this needs to be relocated to the `keyboards/handwired/` folder

Hi zvecr.  Thanks for this review.  I've moved this over to the handwired directory.

File: keyboards/handwired/split89/readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* Full build documented here: [https://github.com/jurassic73/split89](https://github.com/jurassic73/split89)  
+
+* Keyboard Maintainer: [jurassic73](https://github.com/jurassic73)
+* Hardware Supported: ATmega32U4 Pro Micro
+* Hardware Availability: see [Bill of Materials](https://github.com/jurassic73/split89#bill-of-materials).
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make split89/default:default
```
Comment (zvecr) [inline keyboards/handwired/split89/readme.md:16]:
```suggestion
    make handwired/split89/default:default
```

File: keyboards/handwired/split89/rules.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```
Comment (zvecr) [inline keyboards/handwired/split89/rules.mk:5]: stock pro micro? and not an elite c or something?
```suggestion
BOOTLOADER = caterina
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/split89/info.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (zvecr) [inline keyboards/handwired/split89/info.json:1]: This file does not match the defined layout, so is producing the following `qmk info` output (and would also incorrectly render in configurator)
```
Keyboard Name: split89
Manufacturer: jurassic73
Website: 
Maintainer: jurassic73
Keyboard Folder: handwired/split89
Layouts: LAYOUT
Size: 3 x 2
Processor: atmega32u4
Bootloader: atmel-dfu
Layout:
┌──┐┌──┐┌──┐        ┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│k0││k0││k0│        │k0││k0││k0││k0││k0││k1││k1││k1││k1││k1││k1││k1││k1││k1││k1││k2││k2││k2││k2││k2││k2││k2││k2││k2││k2││k3││k3││k3││k3││k3││k3││k3││k3││k3││k3││k4││k4││k4││k4││k4││k4││k4││k4││k4││k4││k5││k5││k5││k5││k5││k5││k5││k5││k5││k5││k6││k6││k6││k6││k6││k6││k6││k6││k6││k6││k7││k7││k7││k7││k7││k7││k7││k7││k7││k7││k8││k8││k8││k8││k8││k8││k8││k8││k8│
└──┘└──┘└──┘        └──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌────┐┌────┐
│k10 ││k12 │
└────┘└────┘
```

File: keyboards/handwired/split89/rules.mk (hunk 2 above)
Comment (jurassic73) [reply keyboards/handwired/split89/rules.mk:5]: Thanks for this.  Yes, this is a stock pro micro unit.  Updated!  :)

File: keyboards/handwired/split89/readme.md (hunk 1 above)
Comment (jurassic73) [reply keyboards/handwired/split89/readme.md]: This has been corrected.  Thanks.

File: keyboards/handwired/split89/info.json (hunk 3 above)
Comment (jurassic73) [reply keyboards/handwired/split89/info.json:1]: Thanks for this.  I will get this sorted.

Comment (jurassic73) [reply keyboards/handwired/split89/info.json:1]: This has been corrected.  Thanks!

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/split89/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    AUD,
+};
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT( \
+        KC_ESC,  KC_F1,  KC_F2,        KC_F3,  KC_F4,  KC_F5,     KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10, KC_F11,     KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS, \
+        KC_1,    KC_2,   KC_3,         KC_4,   KC_5,   KC_6,      KC_7,   KC_8,   KC_9,   KC_0,    KC_MINS, KC_EQL,    KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP, \
+        KC_GRV,  KC_Q,   KC_W,         KC_E,   KC_R,   KC_T,      KC_U,   KC_I,   KC_O,   KC_P,    KC_LBRC, KC_RBRC,   KC_BSLS,   KC_DEL,  KC_END,  KC_PGDN, \
+        KC_TAB,  KC_A,   KC_S,         KC_D,   KC_F,   KC_G,      KC_Y,   KC_J,   KC_K,   KC_L,    KC_SCLN,KC_QUOT,    KC_ENT, \
+        KC_CAPS, KC_Z,   KC_X,         KC_C,   KC_V,   KC_B,      KC_H,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSHIFT, KC_UP, \
```
Comment (fauxpark) [inline keyboards/handwired/split89/keymaps/default/keymap.c:30]:
```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_F1,  KC_F2,        KC_F3,  KC_F4,  KC_F5,     KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10, KC_F11,     KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,
        KC_1,    KC_2,   KC_3,         KC_4,   KC_5,   KC_6,      KC_7,   KC_8,   KC_9,   KC_0,    KC_MINS, KC_EQL,    KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP,
        KC_GRV,  KC_Q,   KC_W,         KC_E,   KC_R,   KC_T,      KC_U,   KC_I,   KC_O,   KC_P,    KC_LBRC, KC_RBRC,   KC_BSLS,   KC_DEL,  KC_END,  KC_PGDN,
        KC_TAB,  KC_A,   KC_S,         KC_D,   KC_F,   KC_G,      KC_Y,   KC_J,   KC_K,   KC_L,    KC_SCLN,KC_QUOT,    KC_ENT,
        KC_CAPS, KC_Z,   KC_X,         KC_C,   KC_V,   KC_B,      KC_H,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSHIFT, KC_UP,
```

File: keyboards/handwired/split89/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        KC_LSFT, KC_LCTL,MO(1),KC_LGUI,KC_LALT,KC_SPACE,    KC_N,   KC_SPACE, KC_RALT,KC_RGUI, KC_APP,  KC_RCTL,   KC_LEFT,   KC_DOWN, KC_RGHT
+        ),
+    /* Volume */
+    [1] = LAYOUT( \
+      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
+      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
+      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, \
+      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS, \
+      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,             KC_VOLU, \
```
Comment (fauxpark) [inline keyboards/handwired/split89/keymaps/default/keymap.c:39]:
```suggestion
    [1] = LAYOUT(
      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,
      KC_TRNS,KC_TRNS,KC_TRNS,      KC_TRNS,KC_TRNS,KC_TRNS,  KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,             KC_VOLU,
```

File: keyboards/handwired/split89/readme.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+* Full build documented here: [https://github.com/jurassic73/split89](https://github.com/jurassic73/split89)  
+
+* Keyboard Maintainer: [jurassic73](https://github.com/jurassic73)
+* Hardware Supported: ATmega32U4 Pro Micro
+* Hardware Availability: see [Bill of Materials](https://github.com/jurassic73/split89#bill-of-materials).
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make /handwired/split89/default:default
```
Comment (fauxpark) [inline keyboards/handwired/split89/readme.md:16]:
```suggestion
    make handwired/split89:default
```

File: keyboards/handwired/split89/readme.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
```
Comment (fauxpark) [inline keyboards/handwired/split89/readme.md:20]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by jurassic73** [comment]
Appreciate the help on this.  :)

**Review by jurassic73 — COMMENTED** [review]
All items resolved that required changes.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by jurassic73** [comment]
@drashna thanks for getting this added to master - exciting to see up there in the tool!

https://config.qmk.fm/#/handwired/split89/LAYOUT

Had a quick question... the default keymap is not visible.  Is this correct?

**Comment by drashna** [comment]
Yup, that has to be added to the qmk configurator repo

**Comment by jurassic73** [comment]
Thanks for the reply... sorry if I'm being dense - is there something for me to do for the default keymap or just wait at this time?

Also, found this typo and submitted a pull request for it.  https://github.com/qmk/qmk_firmware/pull/13231

Appreciate the help!

**Comment by jurassic73** [comment]
> Yup, that has to be added to the qmk configurator repo

Ah okay - gotcha - will get that added here ( https://github.com/qmk/qmk_configurator/tree/master/public/keymaps/h ).  Thanks for the help!

**Pull request merged**